src/libbson/tests/json/bson_corpus/code.json

```diff
@@ -21,12 +21,12 @@
         {
             "description": "two-byte UTF-8 (\u00e9)",
             "canonical_bson": "190000000261000D000000C3A9C3A9C3A9C3A9C3A9C3A90000",
-            "canonical_extjson": "{\"a\" : \"éééééé\"}"
+            "canonical_extjson": "{\"a\" : \"\\u00e9\\u00e9\\u00e9\\u00e9\\u00e9\\u00e9\"}"
```

Comment:
These tests previously passed, but the previous version in this diff was a modified copy of the upstream test.

These tests are now skipped, and I opened [CDRIVER-4017](https://jira.mongodb.org/browse/CDRIVER-4017) to track this change. That issue also talks about re-introducing the original, modified tests, but in a separate directory (related to [CDRIVER-4016](https://jira.mongodb.org/browse/CDRIVER-4016)).